src/Hl7.Fhir.Base/Specification/Terminology/MultiTerminologyService.cs

```diff
@@ -15,19 +15,96 @@ namespace Hl7.Fhir.Specification.Terminology
     /// </summary>
     public class MultiTerminologyService : ITerminologyService
     {
-        private readonly List<ITerminologyService> _services = new();
+        private readonly List<TerminologyServiceRoutingSettings> _termServices = new();
 
         /// <summary>
         /// A collection of multiple terminology services to allow for one or multiple fallback services when validating codes for example.
         /// </summary>
         /// <param name="services">Terminology services to be used. Note that the first service in the list will be used first, the others will be used for fallback based on their order.</param>
-        public MultiTerminologyService(IEnumerable<ITerminologyService> services)
+        public MultiTerminologyService(IEnumerable<ITerminologyService> services) : this(services.Select(s => TerminologyServiceRoutingSettings.CreateDefault(s))) { }
+
+        /// <summary>
+        /// A collection of multiple terminology services to allow for one or multiple fallback services when validating codes for example.
+        /// </summary>
+        /// <param name="settings">Routing settings to be used. You can add settings to prefer certain terminology services, the others will still be used based as fallback based on their order.</param>
+        public MultiTerminologyService(IEnumerable<TerminologyServiceRoutingSettings> settings)
+        {
+            if (settings == null || !settings.Any()) throw Error.ArgumentNull(nameof(settings));
+            _termServices = settings.ToList();
+        }
+
+        /// <summary>
+        /// A collection of multiple terminology services to allow for one or multiple fallback services when validating codes for example.
+        /// </summary>
+        /// <param name="services">Terminology services to be used. Note that the first service in the list will be used first, the others will be used for fallback based on their order.</param>
+        public MultiTerminologyService(params ITerminologyService[] services) : this(services.ToList()) { }
+
+
+        /// <summary>
+        /// A collection of multiple terminology services to allow for one or multiple fallback services when validating codes for example.
+        /// </summary>
+        /// <param name="services">Orderable terminology services to be used. You can set the order of the services to be used using the settings, the others will be used for fallback based on their order.</param>
+        public MultiTerminologyService(params TerminologyServiceRoutingSettings[] services) : this(services.ToList()) { }
+
+
+        /// <summary>
+        /// Add an <see cref="ITerminologyService"/> at the back of the order of terminologyServices to be used.
+        /// </summary>
+        /// <param name="service"><see cref="ITerminologyService"/> to be added.</param>
+        public void AddLast(ITerminologyService service)
+        {
+            var settings = new TerminologyServiceRoutingSettings(service);
```

Comment:
Is there a difference between `TerminologyServiceRoutingSettings.Default(service)` and `new TerminologyServiceRoutingSettings(service)`?  You are using both forms.